react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -210,6 +214,8 @@ class AddPeopleDialog extends Component {
                 this.setState({
                     addToCallInProgress: false
                 });
+
+                return true;
```

Comment:
I'm unsure this will do what you want it to do. The return value of _onSubmit is always true so what happens is invitePeople will get fired then true is returned below and the modal is closed. Returning true here on line 218 is also inconsequential because Dialog does not know what to do with promises so it will not chain onto your invitePeople call, even if your invitePeople call were returned from _onSubmit.